applications/peripheral_sensor_node/README.rst

```diff
@@ -0,0 +1,337 @@
+| P2.06                   | SPI SCK    | Serial Clock    |
++-------------------------+------------+-----------------+
+| P1.14                   | GPIO       | SPI Interrupt   |
++-------------------------+------------+-----------------+
+| P1.11                   | TWI SCL    | TWI Clock       |
++-------------------------+------------+-----------------+
+| P1.12                   | TWI SDA    | TWI Data        |
++-------------------------+------------+-----------------+
+
+Shield is an add-on that you can attach to the nRF54L15 PDK to extend its features and functionalities.
+
+If you want to run the application on the nRF54L15 PDK, you need to attach the sensor shield to it.
+
+.. figure:: /images/peripheral_sensor_node_assy.png
+   :width: 350px
+   :align: center
+   :alt: Peripheral Sensor node assembly
+
+   Peripheral Sensor node assembly
+
+The shield must be oriented in a way that allows its silkscreen to read the same way as PDK.
+The shield must not be rotated.
+
+Power management
+================
+
+To reduce power consumption, the Peripheral Sensor node application mostly remains in the power-down state.
+The application performs burst measurement of the environmental and IMU sensors periodically with a given interval and does additional burst measurement when triggered by the interrupt from the ``ADXL362`` sensor.
+You can configure the periodic burst measurement intervals through the :ref:`CONFIG_APP_CONTROL_BURST_INTERVAL_S <CONFIG_APP_CONTROL_BURST_INTERVAL_S>` Kconfig option.
+By default, the burst interval is set to 120 seconds.
+You can configure the number of measurements for a single burst using the :ref:`CONFIG_APP_CONTROL_BURST_MEAS_CNT <CONFIG_APP_CONTROL_BURST_MEAS_CNT>` Kconfig option, and the time interval between measurements is specified in the :ref:`CONFIG_APP_CONTROL_BURST_MEAS_INTERVAL_MS <CONFIG_APP_CONTROL_BURST_MEAS_INTERVAL_MS>` Kconfig option.
+
+When the application performs burst measurements, it also starts advertising.
+This means it can be connected through a Bluetooth central device, for example, the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+
+Firmware architecture
+=====================
+
+The nRF Peripheral Sensor node application has a modular structure, where each module has a defined scope of responsibilities.
+The application uses the :ref:`app_event_manager` to distribute events between modules in the system.
+
+The following figure shows the application architecture.
+The figure visualizes relations between Application Event Manager, modules, drivers, and libraries.
+
+.. figure:: /images/peripheral_sensor_node_arch.svg
+   :alt: Peripheral Sensor node application architecture
+
+   Peripheral Sensor node application architecture
+
+Programming nRF54L15 PDK
+========================
+
+The nRF54L15 PDK has a J-Link debug IC that can be used to program the firmware.
+
+See the `Building and running`_ section for more information.
```

Comment:
Why is this mentioned here under Overview? This is pretty much a hardware-related information, and we don't list hardware features of the boards in their samples.

I would suggest moving this to under Building and running straight away.

Reply:
The information about J-Link debug IC (since it is a hardware-related information) could be moved to the nRF54L15 getting stared guide, as I couldn't find this information there, and we could remove this section from here. We could consider this in a separate PR, once the guide is available?

"_The nRF54L15 PDK has a J-Link debug IC that can be used to program the firmware._"

Reply:
Yes, I agree, but I would still suggest mentioning this only in the Building and running for now.

Reply:
I removed this from the application documentation.